crates/core/Cargo.toml

```diff
@@ -5,3 +5,12 @@ edition = "2021"
 description = "Extensible interactive autodiff."
 license = "MIT"
 include = ["src/lib.rs"]
+
+[dependencies]
+ts-rs = "6.1"
```

Comment:
Can you change `ts-rs` to be a [dev dependency](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies)?